projects/plugins/wpcomsh/wpcom-migration-helpers/site-migration-helpers.php

```diff
@@ -74,9 +74,8 @@ function aiowp_migration_logging_helper() {
 		'ai1wm_import',
 		function ( $params = array() ) use ( $target_blog_id ) {
 			wpcomsh_record_tracks_event(
-				'wpcom_site_migration_start',
+				'wpcomsh_aiowp_import_start',
```

Comment:
Might it make sense to use the existing `wpcom_import_start` event? Or possibly a new event for imports on Atomic? It feels like we need some way to track all imports on Atomic, so keeping the approach of a generic name with an event prop to identify the tool/platform may be more useful in the long run.

Reply:
Works for me! I've updated to use the existing `wpcom_import_*` events and re-added the migration_tool property.